src/styles/block/card.scss

```
@@ -0,0 +1,130 @@
+.main {
+  @include display-justify-content;
+}
+
+.catalog {
+  width: 1024px;
+  @include display-justify-content;
+  flex-wrap: wrap;
+  padding: 0 40px;
+  margin: 0 auto;
+}
+
+.card {
+  width: 200px;
+  height: 403px;
+  padding: 15px 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid $c-border;
+  border-radius: 5px;
+  &--pos {
+    margin-bottom: 50px;
+  }
+  &__photo {
+    margin: 0 auto 42px;
+  }
+  &__photo-img {
+    box-sizing: content-box;
+    width: 160px;
+    height: 134px;
+    padding-top: 15px;
+    border: 3px solid transparent;
```

Відступи між позиціонуванням і стилем блоку
```suggestion
    padding-top: 15px;
    
    border: 3px solid transparent;
```